examples/components/CCFRAUD_VERTICAL/traininsilo/conda.yaml

```diff
@@ -15,3 +15,4 @@ dependencies:
     - pandas==1.5.2
     - torchmetrics==0.10.3
     - redis==4.5.1
+    - tqdm
```

Comment:
please remove this line, looks like you're not using it in either contributor or host anyway (and it would need to pin down a specific version)